src/App.vue

```diff
@@ -598,25 +598,25 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
   private async validateCompanyInfoPage (): Promise<void> {
     this.setComponentValidate(true)
 
-    // evaluate valid flags. Scroll to invalid components or continue to review.
-    if (await this.validateAndScroll(this.getValidComponentFlags, ComponentFlags)) {
+    // Evaluate valid flags. Scroll to invalid components or continue to review.
+    if (await this.validateAndScroll(this.getValidComponentFlags, AlterationCompanyInfoFlags)) {
       this.setSummaryMode(true)
 
       // Reset global flag
       this.setComponentValidate(false)
 
-      // We don't change views just interchange components, so scroll to top for better UX.
+      // We don't change views, just interchange components, so scroll to top for better UX.
       await this.scrollToTop(document.getElementById('app'))
     }
   }
 
-  /** Perform high level validations before filing. */
-  private async validateApp (): Promise<void> {
+  /** Perform high level component validations before proceeding to filing and paying. */
+  private async validateReviewCertifyPage (): Promise<void> {
```

Comment:
I renamed this as it's specific to alterations. That's not clear from the name but it's better and can be improved later.

Reply:
I didn't like the names either, great change :) 

When i started down this path i didn't really know the end result but now that the picture is a little more clear these changes make perfect sense 👍

Reply:
Like the update to more verbose name that isn't too long either.